web/admin/src/pages/TicketList.svelte

```diff
@@ -164,6 +165,25 @@
   </a>
 </div>
 
+{#if contenders && contenders.length === 0}
+  <EmptyState
+    title="No tickets yet"
+    description="Create tickets with unique registration codes to grant contenders access to your contest."
+  >
+    {#snippet actions()}
+      <wa-button
+        variant="neutral"
+        appearance="accent"
+        onclick={handleOpenCreateDialog}
+        disabled={remainingCodes === undefined || remainingCodes === 0}
+      >
+        <wa-icon slot="start" name="plus"></wa-icon>
+        Create tickets</wa-button
+      >
+    {/snippet}
+  </EmptyState>
+{/if}
+
```

Comment:
```suggestion
```

Remove empty state from this component and put it instead in TicketsListPage.svelte.